proto/fluid_flow.proto

```diff
@@ -0,0 +1,99 @@
+syntax = "proto3";
+
+package smartcore.bos;
+
+option go_package = "github.com/vanti-dev/sc-bos/pkg/gen";
+
+import "google/protobuf/field_mask.proto";
+import "google/protobuf/timestamp.proto";
+import "types/info.proto";
+
+// FluidFlow trait is applicable to devices that control the flow of a fluid (liquid or gas)
+// by opening or closing a passageway. This includes devices such as water valves,
+// gas valves, and other similar control mechanisms used in various industrial and
+// commercial applications.
+service FluidFlowApi {
+  rpc GetFluidFlow(GetFluidFlowRequest) returns (FluidFlow) {}
+  rpc PullFluidFlow(PullFluidFlowRequest) returns (stream PullFluidFlowResponse) {}
+  rpc UpdateFluidFlow(UpdateFluidFlowRequest) returns (FluidFlow) {}
+}
+
+service FluidFlowInfo {
+  rpc DescribeFluidFlow(DescribeFluidFlowRequest) returns (FluidFlowSupport) {}
+}
+
+message FluidFlow {
+  // The target flow rate set at the device e.g. L/s
+  optional float target_flow_rate = 1;
+  // If the flow is powered by a pump or similar device, this field indicates
+  // the frequency at which the drive operates to achieve the desired flow rate.
+  // This is particularly relevant for variable frequency drives (VFDs) used in pumps to control the speed and flow rate.
+  // For devices that do not use a drive frequency to control flow, this field may be omitted or set to a default value of 0 Hz.
+  // The target drive frequency set at the device e.g. Hz
+  optional float target_drive_frequency = 2;
+  // The measured flow rate e.g. L/s
+  // Output only, a zero value indicates that the device does not measure flow rate.
+  optional float flow_rate = 3;
+  // The measured drive frequency e.g. Hz
+  // Output only, a zero value indicates that the device does not measure drive frequency.
+  optional float drive_frequency = 4;
+
+  // FluidFlow Direction describes the control of fluid flow, is returning the fluid passed through or is blocking the flow.
```

Comment:
This comment isn't great english. Looks to have remnants of past comments and "is returning the fluid passed through" doesn't really make sense.